ethportal-peertest/src/scenarios/eth_rpc.rs

```diff
@@ -25,50 +27,11 @@ pub fn test_eth_get_block_by_hash(_peertest_config: PeertestConfig, peertest: &P
         method: "eth_getBlockByHash".to_string(),
         id: 11,
         params: Params::Array(vec![
-            json!("0x55b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286"),
+            json!("0x6251d65b8a8668efabe2f89c96a5b6332d83b3bbe585089ea6b2ab9b6754f5e9"),
             json!(false),
         ]),
     };
 
     let result = make_ipc_request(&peertest.nodes[0].web3_ipc_path, &request).unwrap();
-    assert_eq!(json!(15537393), result["number"]);
-}
-
-pub fn test_eth_get_block_by_number(_peertest_config: PeertestConfig, peertest: &Peertest) {
```

Comment:
I removed this test because the epoch accumulator was causing strange bugs, due to it's large size. A potential cause for this error could be our current jsonrpc server implementation, since the error came from `jsonrpc/service.rs`. I spent some time debugging, but didn't consider it to be worthwhile pursuing. Since @ogenev plans to remove support for these two scenarios, I figured we can just remove this test for now - and it will be re-introduced once we have an improved jsonrpc server / support for this endpoint

Reply:
Can you add a tracking issue?

Reply:
#524